docs/source/docker/advanced/active_learning.rst

```diff
@@ -1,7 +1,294 @@
 .. _ref-docker-active-learning:
 
-Active learning
-==============================================
+Active Learning
+===============
+
+Lightly makes use of active learning scores to select the samples which will yield
+the biggest improvements of your machine learning model. The scores are calculated
+on-the-fly based on model predictions and provide the selection algorithm with feedback
+about the uncertainty of the model for the given sample. 
+
+Learn more about the concept of active learning scores:
+:ref:`lightly-active-learning-scorers`.
+
+Prerequisites
+--------------
+In order to do active learning with Lightly, you will need the following things:
+
+- The installed Lightly docker (see :ref:`ref-docker-setup`)
+- A dataset with a configured datasource (see :ref:`ref-docker-with-datasource-datapool`)
+
+.. note::
+
+    The dataset does not need to be empty! For example, an initial selection without
+    active learning can be used to train a model. The predictions from this model
+    can then be used to improve your dataset by adding new images to it through active learning.
+
+
+Next, you need to prepare the model predictions in the correct format.
+
+Predictions
+-----------
+
+In the following, we will outline the format of the predictions required by the 
+Lightly docker. Everything regarding predictions will take place in a subdirectory
+of your configured datasource called `.lightly/predictions`.
+
+Prediction Tasks
+^^^^^^^^^^^^^^^^
+To let Lightly know what kind of prediction tasks you want to work with, Lightly
+needs to know their names. It's very easy to let Lightly know which tasks exist:
+simply add a `tasks.json`` in your storage bucket stored at the subdirectory `.lightly/predictions/`.
+
+The `tasks.json` file must include a list of your task names which must match name
+of the subdirectory where your prediction schemas will be located.
+
+.. note::
+
+    Only the task names listed within `tasks.json`` will be considered.
+    Please ensure that the task name corresponds with the location of your prediction schema.
+
+.. code-block:: javascript
+    :caption: .lightly/predictions/tasks.json
+
+    [
+        "my_classification_task",
```

Comment:
I guess here we have something missing? JSON is key: value, no?

Reply:
Here I'm not sure why I have to add this file. I just list the directories I actually want to use for predictions and which ones to ignore?

E.g. if I have the following folder structure
```
predictions/
`-my_classification_task/
`-my_object_detection_task/
`-some_random_folder_within_predictions_I_dont_want_to_use/
```

I create the `task.json` just to tell lightly that we can ignore `predictions/some_random_folder_within_predictions_I_dont_want_to_use`?

If this is the case I would make a more concrete example (maybe with a folder we want to ignore) and also mention what happens if we accidentally list that folder.

Reply:
> I guess here we have something missing? JSON is key: value, no?
No, it can just be a list.

> If this is the case I would make a more concrete example (maybe with a folder we want to ignore) and also mention what happens if we accidentally list that folder.
👍